src/containers/MyNdla/Folders/PreviewFoldersPage.tsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2023-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { useCallback, useContext, useMemo, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import FolderActions from './FolderActions';
+import FolderButtons from './FolderButtons';
+import { ViewType } from './FoldersPage';
+import { AuthContext } from '../../../components/AuthenticationContext';
+import { STORED_RESOURCE_VIEW_SETTINGS } from '../../../constants';
+import { GQLFolder, GQLFoldersPageQuery } from '../../../graphqlTypes';
+import { useGraphQuery } from '../../../util/runQueries';
+import SharedFolderPage from '../../SharedFolderPage/SharedFolderPage';
+import MyNdlaPageWrapper from '../components/MyNdlaPageWrapper';
+import { foldersPageQuery, useFolder } from '../folderMutations';
+
+const PreviewFoldersPage = () => {
+  const { folderId } = useParams();
+  const { examLock } = useContext(AuthContext);
+  const selectedFolder = useFolder(folderId);
+  const [viewType, _setViewType] = useState<ViewType>(
+    (localStorage.getItem(STORED_RESOURCE_VIEW_SETTINGS) as ViewType) || 'list',
+  );
+  const { data } = useGraphQuery<GQLFoldersPageQuery>(foldersPageQuery);
+
+  const folders: GQLFolder[] = useMemo(
+    () =>
+      selectedFolder
+        ? selectedFolder.subfolders
+        : (data?.folders as GQLFolder[]) ?? [],
+    [selectedFolder, data?.folders],
+  );
+  const [_focusId, setFocusId] = useState<string | undefined>(undefined);
+
+  const setViewType = useCallback((type: ViewType) => {
+    _setViewType(type);
+    localStorage.setItem(STORED_RESOURCE_VIEW_SETTINGS, type);
+  }, []);
+
+  const dropDownMenu = useMemo(
+    () => (
+      <FolderActions
+        selectedFolder={selectedFolder}
+        setFocusId={setFocusId}
+        folders={folders}
+        inToolbar
+      />
+    ),
+    [selectedFolder, folders, setFocusId],
+  );
+
+  const folderButtons = useMemo(
+    () => (
+      <FolderButtons selectedFolder={selectedFolder} setFocusId={setFocusId} />
+    ),
+    [selectedFolder, setFocusId],
+  );
```

Comment:
For fokus sin del tror jeg det å ha alle tilgangene til alle knappene og menyen vil være et styr å fikse.

For meg så er vi på en helt ny side her og ikke lenger i konteksten hvor man kan redigere og tulle med mappen, så mappe funksjonaliteten/verktøyene gir ikke mening å ha her.  Dette er en preview så den skal jo være så lik som mulig en delt mappe, og i den offentlige siden til en delt mappe så har du jo ikke tilgang til slike verktøy. Så tipper du kan bare fjerne alle props fra `MyNdlaPageWrapper`, utenom `viewType` propene

Reply:
Tja. Jeg mener at sida previewen skjer internt, bør man fortsatt kunne behandle sin mappe (gjorde noen småtester for å se at jeg f.eks. kunne sette mappa som delt mens jeg var i forhåndsvisninga). Dvs mappe-innholdet er jo forhåndsvist, men i kontekst av en innlogga bruker? Eller er det dumt?
Tenker f.eks. hvis man er inne på en forhåndsvisning, også vil over på en annen del av MinNDLA. Så er det litt kjedelig om man må gå tilbake først (et ekstra klikk)?

Reply:
Toolbar/menubar kan være der, men tenker folderbuttons og folderActions.